docs/source/whats_new.rst

```diff
@@ -1,3 +1,21 @@
+What's New in cartopy 0.16
+==========================
+
+:Release: 0.16.0
+:Date: 14th December 2017
```

Comment:
We weren't planning on releasing cartopy then, and indeed we didn't. Could you leave this line blank please?